dimod/binary/binary_quadratic_model.py

```diff
@@ -1578,19 +1618,34 @@ def iter_neighborhood(self, v: Variable) -> Iterator[Tuple[Variable, Bias]]:
         return self.data.iter_neighborhood
 
     def iter_neighbors(self, u):
-        warnings.warn('bqm.iter_neighbors(v) is deprecated since dimod 0.10.0.'
+        """Deprecated
+
+        .. deprecated:: 0.10.0
+
+            ``BinaryQuadraticModel.iter_neighbors()`` will be removed in dimod 0.12.0.
+            Use ``(v for v, _ in bqm.iter_neighborhood(v))`` instead.
+
+        """
+        warnings.warn('bqm.iter_neighbors(v) is deprecated since dimod 0.10.0 '
+                      'and will be removed in 0.12.0.'
                       ' Use (v for v, _ in bqm.iter_neighborhood(v)) instead.',
                       DeprecationWarning, stacklevel=2)
         for v, _ in self.iter_neighborhood(u):
             yield v
 
     def iter_quadratic(self, variables=None) -> Iterator[Tuple[Variable, Variable, Bias]]:
-        """Iterate over the quadratic biases"""
+        """Iterate over the quadratic biases.
+
+        .. deprecated:: 0.10.0
+
+            Argument ``variables`` will be removed in dimod 0.12.0.
```

Comment:
That's what's deprecated.

Reply:
```
warnings.warn('passing variables to bqm.iter_quadratic() is '
                          'deprecated since dimod 0.10.0. Use '
                          'deprecated since dimod 0.10.0 and will be removed in 0.12.0. Use '
                          'bqm.iter_neighborhood(v) instead.',
```